test/models/tf_ov_model_zoo/run_infer_single.sh

```diff
@@ -57,83 +77,174 @@ function get_model_repo {
-        (( count == 0 )) && first_infer_time=$i
+        (( count < $num_warmup_iters )) && warmup_iters_time=$(echo $warmup_iters_time+$i | bc )
         ((count++))
     done
-    (( count > 1 )) && total=$(echo $total-$first_infer_time | bc )
+    (( count > $num_warmup_iters )) && total=$(echo $total-$warmup_iters_time | bc )
     avg=$(echo "scale=4; $total / $count" | bc)
     avg=$( printf %.04f $avg )
     echo $avg
 }
 
+function run_bench_stocktf {
+    pushd . >/dev/null
+    cd ${LOCALSTORE}/demo
+    TMPFILE=${WORKDIR}/tmp_output$$
+    ./run_infer.sh ${MODEL} ${IMGFILE} $NUM_ITER "tf" $device 2>&1 > ${TMPFILE}
+    ret_code=$?
+    if (( $ret_code == 0 )); then
+        echo
+        echo "Stock Tensorflow: Checking inference result (warmups=$WARMUP_ITERS) ..."
+        ret_code=1
+        INFER_PATTERN=$( echo $INFER_PATTERN | sed -e 's/"/\\\\"/g' )
+        grep "${INFER_PATTERN}" ${TMPFILE} >/dev/null && echo "TEST PASSED" && ret_code=0
+        print_infer_times $NUM_ITER $WARMUP_ITERS "${TMPFILE}"
+        INFER_TIME_STOCKTF=$INFER_TIME
+    fi
+    echo
+    rm ${TMPFILE}
+    popd >/dev/null
+}
+
+function run_bench_stockov {
+    pushd . >/dev/null
+    VENVTMP="$WORKDIR/venv_temp_stockov" # to ensure no side-effects of any pip installs
+    [ -d $VENVTMP ] && rm -rf $VENVTMP
+    virtualenv -p python3 $VENVTMP
+    source $VENVTMP/bin/activate
+    pip list | grep 'opencv-python' 2>&1 >/dev/null; (($?==0)) || pip install opencv-python;
+    pip list | grep 'openvino' 2>&1 >/dev/null; (($?==0)) || pip install openvino;
+
+    cd ${LOCALSTORE}/demo
+    TMPFILE=${WORKDIR}/tmp_output$$
+    pythonlib=$(echo $(dirname $(which python3))/../lib)
+    echo LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$pythonlib ./run_ov_infer.sh ${MODEL} ${IMGFILE} $NUM_ITER $device
+    LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$pythonlib \
+        ./run_ov_infer.sh ${MODEL} ${IMGFILE} $NUM_ITER $device 2>&1 > ${TMPFILE}
+    ret_code=$?
+    if (( $ret_code == 0 )); then
+        echo
+        echo "Stock OpenVINO: Checking inference result (warmups=$WARMUP_ITERS) ..."
+        print_infer_times $NUM_ITER $WARMUP_ITERS "${TMPFILE}"
+        INFER_TIME_STOCKOV=$INFER_TIME
+    fi
+    echo
+    deactivate # venv
+    rm ${TMPFILE}
+    rm -rf $VENVTMP
+    popd >/dev/null
+}
+
 ################################################################################
 ################################################################################
 
-pip list | grep 'Pillow' 2>&1 >/dev/null; found=$(( ! $? ));
-if (( ! $found )); then pip install Pillow; fi
+pip list | grep 'Pillow' 2>&1 >/dev/null; (($?==0)) || pip install Pillow;
+if [ "${BENCHMARK}" == "YES" ]; then
+    pip list | grep 'networkx' 2>&1 >/dev/null; (($?==0)) || pip install networkx;
+    pip list | grep 'defusedxml' 2>&1 >/dev/null; (($?==0)) || pip install defusedxml;
+    pip list | grep 'test-generator' 2>&1 >/dev/null; (($?==0)) || pip install test-generator==0.1.1;
+fi
```

Comment:
Are these OV dependencies? I think that it's better to explicitly install them up-front in the manifest.

Also, please use something like `grep -E "^pkg-name "` for a proper match.

Reply:
Not sure, how to install in manifest. Please suggest. Will update the grep pattern.

Reply:
I meant that the install should be done in the "build" step of the `bench-cpu.yml` file. But it's not a big deal to have it here too as a sanity check. Your call.